omnigibson/simulator.py

```diff
@@ -202,7 +202,13 @@ def _set_physics_engine_settings(self):
         # collide with each other, and modify settings for speed optimization
         self._physics_context.set_invert_collision_group_filter(True)
         self._physics_context.enable_ccd(gm.ENABLE_CCD)
-        self._physics_context.enable_flatcache(gm.ENABLE_FLATCACHE)
+
+        if hasattr(self._physics_context, "enable_fabric"):
```

Comment:
same here, we should just have a programmtic version check instead of a blind if/else clause so it's more interpretable

Reply:
we can probably use the `meets_minimum_version` API I recently added to `omnigibson.utils.python_utils`

Reply:
neat API, done!